lib/install/config/webpack/dev_server.yml

```diff
@@ -0,0 +1,4 @@
+# Restart webpack-dev-server if you make changes here
```

Comment:
Seems like we should standardize naming. For js, we're going with "development.server.js" and this is "dev_server.yml". I'd be fine with "development.server.yml"